Teensy/src/gpio/gpio.cpp

```diff
@@ -11,59 +11,59 @@
 #include <Arduino.h>
 #include "gpio.h"
 
-static int32_t gpio_getWheelSpeed();
-static uint32_t gpio_getEngineRevs();
-static int32_t gpio_getDamperPosition();
-static int32_t gpio_getGearPosition();
-static uint32_t gpio_getSteeringWheelPosition();
-static int32_t gpio_getStrain();
+static daq_WheelSpeed_t gpio_getWheelSpeed();
+static daq_EngineRev_t gpio_getEngineRevs();
+static daq_DamperPos_t gpio_getDamperPosition();
+static daq_GearPos_t gpio_getGearPosition();
+static daq_SteeringWhlPos_t gpio_getSteeringWheelPosition();
+static daq_Strain_t gpio_getStrain();
 static GyroData_s gpio_getGyro();
-static uint32_t gpio_getVBat();
-static uint32_t gpio_getThrottlePosition();
-static uint32_t gpio_getFuelPressure();
+static daq_BatteryV_t gpio_getVBat();
+static daq_ThrottlePos_t gpio_getThrottlePosition();
+static daq_FuelPressure_t gpio_getFuelPressure();
 
 bool updateSensorInfo(sensorData_s *pSensorData)
 {
-    pSensorData->wheel_speed = gpio_getWheelSpeed();
-    pSensorData->engine_revs = gpio_getEngineRevs();
-    pSensorData->damper_position = gpio_getDamperPosition();
-    pSensorData->gear_position = gpio_getGearPosition();
-    pSensorData->steering_wheel_position = gpio_getSteeringWheelPosition();
+    pSensorData->wheelSpeed_mph = gpio_getWheelSpeed();
+    pSensorData->engineRev_rpm = gpio_getEngineRevs();
+    pSensorData->damperPos_mm = gpio_getDamperPosition();
+    pSensorData->gearPos = gpio_getGearPosition();
+    pSensorData->steeringWheelPos_degrees = gpio_getSteeringWheelPosition();
     pSensorData->strain = gpio_getStrain();
     pSensorData->gyro = gpio_getGyro();
-    pSensorData->VBat = gpio_getVBat();
-    pSensorData->throttle_position = gpio_getThrottlePosition();
-    pSensorData->fuel_pressure = gpio_getFuelPressure();
+    pSensorData->batteryVoltage_dV = gpio_getVBat();
```

Comment:
why 'dV', not just 'V'?

Reply:
I set it to dV for deci-volts as you put that as the units in a comment in `dap_common.h`

Reply:
Ahh I completely forgot about that, but makes sense as we're using integers